packages/container/README.md

```diff
@@ -0,0 +1,20 @@
+# @kaizen/container
+
+Container combines all the boilerplate and setup to ensure Kaizen components are consistent across different codebases. This includes:
+* Theme management
+* Browser normalisation 
+* Fonts
+* CSS Polyfills
```

Comment:
Crazy idea: We could use this component to cancel out global styles.

Say performance-ui has this annoying code (it does):

```
figure {
  margin: 1em 40px;
}
```

We could have our container component include a stylesheet called:

```
.overrideGlobalStyles {
    figure {
        margin: initial;
    }
}
```

These rules would have higher specificity than our annoying global styles, and would "reset" them for any child components. Assuming all Kaizen stuff renders inside this container in future, we now have a single place to cancel our global styles.

We could then go through murmur and perf-ui, build a comprehensive list of global styles, and reset them all here, and be done with that class of bug.

Reply:
This would definitely resolve this scope of issue for us in the long term, however I'm quite nervous about rolling this out in the first iteration as it will likely cause some unintended regressions.

Reply:
I'd be keen to just get this merged.

If we did it we could also scope it with some props:

```
<Container globalStyleResets="murmur" />
```

... which makes me wonder if we can't just let the consumers (perf-ui and murmur are the only culprits) handle this in a custom component in their own codebase.

```
<Container>
   <MurmurGlobalStyleReset>
        {...}
    </MurmurGlobalStyleReset>
</Container>
```

Reply:
Then the problems become entirely theirs to handle